ipaserver/install/server/upgrade.py

```diff
@@ -1710,7 +1710,7 @@ def upgrade_configuration():
             )
         upgrade_pki(ca, fstore)
 
-        if kra.is_configured():
+        if kra.is_installed():
```

Comment:
18 lines below there's a very similar check, you should probably change that one, too.

Reply:
Good catch, done.